staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2836,6 +2836,10 @@ type InterfaceBindingPlugin struct {
 	// version: v1alphav1
 	// +optional
 	DownwardAPI NetworkBindingDownwardAPIType `json:"downwardAPI,omitempty"`
+	// ComputeResourceOverhead specifies the resource overhead that should be added to the compute container when using the binding.
+	// version: v1alphav1
+	// +optional
+	ComputeResourceOverhead k8sv1.ResourceList `json:"computeResourceOverhead,omitempty"`
```

Comment:
I would consider here using `ResourceRequirements` to be in sync with the pod/container structure.
It does make sense to add `limits` for example.
I am unsure about `claims`.

Reply:
Done.